src/main/riddl/LocationsEventsProjections/locationsEventsProjections.riddl

```diff
@@ -0,0 +1,71 @@
+context LocationEventsProjections is  {
+  include "locationsEventsMessages"
+
+  projection LocationsEventsView is {
+     fields {
+        orgId: OrganizationId,
+        locationId: LocationId,
+        capacity: Integer,
+        reservationId: ReservationId,
+        reservationStart: TimeStamp,
+        reservationEnd: TimeStamp,
+        isPrivate: Boolean // we can use this for a sum of location, reservation, & venue privacy
```

Comment:
I don't entirely understand the comment here. But, it makes me nervous. When you say a sum of location, reservation, and venue privacy, are you suggesting anding the isPrivate flags from all three of these entities in the view? If that is the case it makes me wonder what the purpose of this projection is. What queries are we anticipating to execute against this projection?

To me the reason to have a view that joins locations and events is to derive a list of events grouped by location. Since we are looking at list of events I would primarily be interested in which events in that list are public and which are private. The isPrivate flag for reservation and venue would be irrelevant. The isPrivate flag for a venue is used limit which organizations can schedule the venue. I need to review reservation. I am not sure whether or not reservation needs an isPrivate flag. A private event limits which members have visibility and who can attend.

Reply:
That makes sense, I don't think I thought this logic through entirely. Thanks!

Reply:
This projection was built for the EasyButton story - it's supposed to take in EventInfo and spit out Locations (or, as I thought, a new Reservation). So it is misnamed - should be LocationsReservationsProjection. Sorry for the confusion.